bytecomp/simplif.mli

```diff
@@ -13,7 +13,11 @@
 (*                                                                        *)
 (**************************************************************************)
 
-(** Lambda simplification and lambda plugin hooks *)
+(** Lambda simplification and lambda plugin hooks
+
+    Plugins: beware this module makes no compatibility guarantees.
```

Comment:
I'd say "beware, this module is not guaranteed compatible between releases." to make the meaning clearer. (The comma is also needed.)